versioned_docs/version-2.0/how_to_guides/playground/custom_tls_certificates.mdx

```diff
@@ -0,0 +1,28 @@
+---
+sidebar_position: 1
+---
+
+# Use custom TLS certificates
+
+:::note Self-hosted enterprise only (version 0.6 and later)
+
+This feature is only available for self-hosted enterprise customers, starting from version 0.6.
+
+If you are interested in this plan, please contact sales@langchain.dev for more information.
+
+:::
+
+You can use custom TLs certificates to connect to model providers in the LangSmith playground. This is useful if you are using a self-signed certificate or a certificate from a custom certificate authority.
+
+To use custom TLS certificates, you need to set the following environment variables. See the [self hosted deployment section](../../category/self-hosting) for more information on how to set up application configuration.
+
+- `LANGSMITH_PLAYGROUND_TLS_MODEL_PROVIDERS`: A comma-separated list of model providers that require custom TLS certificates. Note that `azure_openai` is currently the only supported model provider that supports custom TLS certificates, but more providers will be supported in the future.
```

Comment:
should we perhpas obfuscate this(in the code) while its the only supported provider? i didnt realize we only supported it in azure_openai

Reply:
wdym by obfuscate?